1-exercises/B-while-loop/exercise.js

```diff
@@ -6,9 +6,22 @@
 */
 
 function evenNumbers(n) {
-    // TODO
+    let evenNumbersUntiln=0;
+    let i=1;
+    
+    while(i < n){
+        evenNumbersUntiln = evenNumbersUntiln + "," + 2 * i;
+        i++;
+    }
+
+    if(n !== 0){
+        return evenNumbersUntiln;
+    } else {
+        return "nothing";
+    }
+
```

Comment:
Good use of a check - you could make it n>0 so that only positive numbers are accepted.

Reply:
Thank you